Thyme/Thyme.js

```
@@ -0,0 +1,311 @@
+        }
+
+        const dataView = new DataView(buffer);
+        const handler = Thyme.#writeHandlers[type];
+        if (handler) {
+            return handler(dataView, posRef, data);
+        }
+    }
+
+    static read(type, buffer, posRef, rest) {
+        if (Array.isArray(buffer) && buffer.every(Number.isInteger)) {
+            buffer = new Uint8Array(buffer).buffer;
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)